server/sonar-db-dao/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -414,6 +414,15 @@
       </if>
     </where>
   </select>
+  
+  <select id="selectUuidsByKeyFromProjectKey" parameterType="string" resultType="KeyWithUuid">
+    SELECT
+     p.kee, p.uuid
+    FROM 
+      projects p
+    INNER JOIN 
```

Comment:
I know the join is simple, but factoring SQL of `selectUuidsByKeyFromProjectKey` and `selectAllComponentsFromProjectKey` with a SQL import would have enforced that both method SQL queries keep in sync but for the returned columns

Reply:
The other query was dropped